Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -403,15 +436,16 @@ static void __GetGlyphRunDetails(_CTTypesetter* ts, CFRange range, CGRect frameS
 
             // TODO::
             // This is a temp workaround until we can have actual glyph origins
-            run->_glyphOrigins.emplace_back(CGPoint{ xPos, yPos });
-            for (int index = 0; index < glyphRunDetails._dwriteGlyphRun[i].glyphCount - 1; index++) {
-                run->_glyphOrigins.emplace_back(CGPoint{ xPos + glyphRunDetails._dwriteGlyphRun[i].glyphAdvances[index], yPos });
+            for (int index = 0; index < glyphRunDetails._dwriteGlyphRun[i].glyphCount; index++) {
+                run->_glyphOrigins.emplace_back(CGPoint{ xPos, yPos });
+                run->_glyphAdvances.emplace_back(CGSize{ glyphRunDetails._dwriteGlyphRun[i].glyphAdvances[index], 0.0f });
```

Comment:
The offset is always 0.0, but so is the yPos, despite it very much not always being 0...

---
In reply to: [79692524](https://github.com/Microsoft/WinObjC/pull/989#discussion_r79692524) [](ancestors = 79692524)